### DISCUSSION TITLE
Feature request: Placeholder as a widget

### DISCUSSION THREAD

**Discussion by tommienu** [singerdmx/flutter-quill#394] [Feature Request]
Any thoughts on adding placeholders as a widget rather than String? It's quite common to have separate padding or to center placeholders in textfields.

Eg.
![image](https://user-images.githubusercontent.com/1025989/133144146-cbfab196-0a92-4e26-836c-d4acdf0027b6.png)

While this looks more balanced (photoshopped):
![image](https://user-images.githubusercontent.com/1025989/133144618-e31eb222-81f7-429b-8c1f-7154af95acbc.png)

Maybe add an additional placeholderContainer parameter which will accept a widget and require the placeholder string to be set?

Food for thought, love the lib!

**Comment by singerdmx** [comment]
You can submit a PR for it
